errors/no-document-viewport-meta.md

```diff
@@ -0,0 +1,44 @@
+# Viewport `meta` tags should not be used in `_document.js`'s `<Head>`
+
+#### Why This Error Occurred
+
+Adding `<meta name="viewport" ...>` in `pages/_document.js` will lead to unexpected results since the viewport is handled by `next/head`.
```

Comment:
```suggestion
Adding `<meta name="viewport" ...>` in `pages/_document.js` will lead to unexpected results since it cannot be deduped.
The viewport tag should be handled by `next/head` in `pages/_app.js`.
```